File: CodeForces/ProblemSet/800/Draft/734A.js
function solve(n, s) {
  let a = 0;
  let d = 0;
  for (let i = 0; i < n; i++) {
    if (s[i] === "A") a++;
    else d++;
  }
  if (a === d) return "FRIENDSHIP";
  return a > d ? "Anton" : "Danik";
}

function main() {
  const n = Number(readline());
  const s = readline();
  console.log(solve(n, s));
}

main();

File: CodeForces/ProblemSet/800/734A.py
def solve(s):
    a, d = 0, 0
    for win in s:
        if win == "A":
            a += 1
        else:
            d += 1
    if a == d:
        return "Friendship"
    return "Anton" if a > d else "Danik"


def main():
    n, s = input(), input()  # Ignore n
    print(solve(s))


main()
